download.sh

```diff
@@ -298,16 +297,19 @@ hiera_include('classes')
 EOF
 
 # Ansible
-
-if [ -n "$ansiblegit" ]; then
-    clone "$ansiblegit" ansible
-    mkdir -p $TOP/etc/ansible
-    if [ -z "$stable" ]; then
-        echo "Please indicate to download.sh stable=<latest stable release>."
-    else
-        cp -a ansible/upgrade/$stable/$tag/* $TOP/etc/ansible/
-    fi
-fi
+mkdir -p $TOP/etc/ansible
+for p in $PROFILES; do
+  mkdir -p $TOP/etc/ansible/roles/$p/tasks $TOP/etc/ansible/roles/$p/files
+  cp infra/upgrade/files/* $TOP/etc/ansible/roles/$p/files
+  cat infra/upgrade/snippets/edeploy.yaml > $TOP/etc/ansible/roles/tasks/${p}.yaml
```

Comment:
Shouldn't it be $TOP/etc/ansible/roles/${p}/tasks/main.yaml instead of $TOP/etc/ansible/roles/tasks/${p}.yaml ?